src/triagebot/no-merge.md

```diff
@@ -39,7 +39,7 @@ There are three optional values that can be specified in the table:
 > 
 > ```shell-session
 > $ # rebase
-> $ git rebase -i master
+> $ git pull --rebase https://github.com/rust-lang/rust.git master
 > $ # delete any merge commits in the editor that appears
```

Comment:
this comment is outdated with the removal of -i

Reply:
removing the merges is probably unnecessary so maybe just remove

Reply:
Maybe we should restate to "fix conflicts"?

Reply:
@Kobzol Yea, maybe it would be better to just remove this comment (particularly since triagebot doesn't include anything). I think most users should be able to figure out what git says when there is a conflict.

Reply:
Comment removed.